docs/documentation/siddhi-4.0.md

```diff
@@ -1199,43 +1190,43 @@ insert into StateNotificationStream;
 
 ### Output rate limiting
 
-Output rate limiting allows queries to emit events periodically based on the condition specified.
+Output rate limiting allows queries to output events periodically based on a specified condition.
 
 **Purpose**
 
-This allows you to limit the output to avoid overloading following executions and to remove unnecessary information.
+This allows you to limit the output to avoid overloading the subsequent executions, and to remove unnecessary information.
 
 **Syntax**
 
-The following is the syntax of an output rate limiting configuration.
+The syntax of an output rate limiting configuration is as follows:
 
 ```sql
 from <input stream> ...
 select <attribute name>, <attribute name>, ...
 output <rate limiting configuration>
 insert into <output stream>
 ```
-Three type of output rate limiting configuration are available, such as 
+Siddhi supports three types of output rate limiting configurations as explained in the following table: 
 
 Rate limiting configuration|Syntax| Description
 ---------|---------|--------
-Based on time | `<output event> every <time interval>` | Output `<output event>` for every given `<time interval>` time interval.
-Based on number of events | `<output event> every <event interval> events` | Output `<output event>` for every `<event interval>` number of events.
-Snapshot based output | `snapshot every <time interval>`| Output all events in the window (or the last event if no window is defined in the query) for every given `<time interval>` time interval.
+Based on time | `<output event> every <time interval>` | This outputs `<output event>` every `<time interval>` time interval.
+Based on number of events | `<output event> every <event interval> events` | This outputs `<output event>` for every `<event interval>` number of events.
+Snapshot based output | `snapshot every <time interval>`| This outputs all events in the window (or the last event if no window is defined in the query) for every given `<time interval>` time interval.
 
-Here the `<output event>` specifies which event(s) should be emitted as the output of the query. 
+Here the `<output event>` specifies the event(s) that should be returned as the output of the query. 
 The possible values are as follows:
-* `first` : Only the first event processed by the query in the specified time interval/sliding window is emitted.
-* `last` : Only the last event processed by the query in the specified time interval/sliding window is emitted.
-* `all` : All the events processed by the query in the specified time interval/sliding window are emitted. **When no `<output event>` is defined `all` will be used as default.**
+* `first` : Only the first event processed by the query during the specified time interval/sliding window is output.
```

Comment:
I think its clear if we say "emitted" then output for `first` `last` and `all`

Reply:
Changed